pkg/internal/api_helpers.go

```diff
@@ -165,12 +168,30 @@ func (spc *StatusPageComponentChange) getLabels() map[string]string {
 	return map[string]string{"component": *spc.Component, "monitor": *spc.MonitorLogicalName}
 }
 
+// Map statuses to numeric values for Frames
 func spcStatusToInt(status string) int8 {
 	statuses := map[string]int8{
-		"up":          0,
-		"operational": 0,
-		"degraded":    1,
-		"down":        2,
+		"up":                   0,
+		"operational":          0,
+		"Good":                 0,
+		"Information":          0,
+		"NotApplicable":        0,
+		"Advisory":             0,
+		"Healthy":              0,
+		"available":            0,
+		"information":          0,
+		"Degraded":             1,
+		"Warning":              1,
+		"degraded":             1,
+		"disruption":           1,
+		"down":                 2,
+		"Disruption":           2,
+		"Critical":             2,
+		"outage":               2,
+		"degraded_performance": 2,
+		"major_outage":         2,
+		"partial_outage":       2,
+		"under_maintenance":    3,
 	}
 	result := statuses[status]
```

Comment:
Should we set a default value incase a new status is added

Reply:
I believe since this is a value type it will default to 0 in that case